### PR TITLE
added html and content

### PR DESCRIPTION
**Pull request by vherow**

### CODE REVIEW

**Review by axamei — CHANGES_REQUESTED** [review]
Please add appropriate html elements to the all pages based on your wireframes.

**Comment by vherow** [comment]
hello. i added all the html content that was needed.

File: portfolio.html [hunk 1]
```diff
@@ -18,7 +19,24 @@
     </nav>
   </header>
   <main>
+    <table width="400" cellpadding="10" cellspacing="5"
```
Comment (axamei) [inline portfolio.html]: Tables are only for data, not layout. Please adjust this code to use valid html.

File: index.html [hunk 2]
```diff
@@ -20,9 +21,10 @@
...
     <div> <img src="images/smiling dog.jpg" alt="a puppy that is smiling"</div>
     <div>
-      <h1>Section Title</h1>
+      <h2>About Victoria</h2>
+      <p>Information about Victoria</p>
     </div>
   </main>
 </body>
-<footer></footer>
+<footer> (c) 2019 Victoria Herow</footer>
```
Comment (axamei) [inline index.html]: You can use `&copy;` to get the copyright symbol.

**Review by axamei — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (vherow) [reply index.html]: all set. thanks!

File: portfolio.html (hunk 1 above)
Comment (vherow) [reply portfolio.html]: ok, thanks. i used an unordered list instead.

**Pull request merged**